spec/support/features/clearance_helpers.rb

```diff
@@ -0,0 +1,53 @@
+module Features
+  module ClearanceHelpers
+    def reset_password_for(email)
+      visit new_password_path
+      fill_in 'password_email', with: email
+      click_link_or_button I18n.t('helpers.submit.password.submit')
+    end
+
+    def sign_in
+      password = 'password'
+      user = FactoryBot.create(:user, password: password)
+      sign_in_with user.email, password
+    end
+
+    def sign_in_with(email, password)
+      visit sign_in_path
+      fill_in 'session_email', with: email
+      fill_in 'session_password', with: password
+      click_button(I18n.t('helpers.submit.session.submit'), match: :first)
```

Comment:
This will help make rubocop happy and ensure that you are finding the button in the form rather than the navbar

```suggestion
      within('#clearance') do
        fill_in 'session_email', with: email
        fill_in 'session_password', with: password
        click_link_or_button I18n.t('helpers.submit.session.submit')
      end
```